src/utils/loadable.jsx

```diff
@@ -0,0 +1,40 @@
+import { PureComponent } from 'react';
+import Loadable from 'react-loadable';
+import { withStyles } from '@material-ui/core/styles';
+import Spinner from '../components/Spinner';
+
+@withStyles(theme => ({
+  view: {
+    textAlign: 'center',
+    margin: 100,
+    backgroundColor: theme.palette.background.default,
+    height: 'calc(100% - 60px)',
+    marginTop: 60,
+  },
+}))
+class Loading extends PureComponent {
+  content() {
+    const { error, timedOut, pastDelay } = this.props;
+
+    if (error) {
+      throw error;
+    } else if (timedOut || pastDelay || !error) {
```

Comment:
I don't think we need `|| !error` since the if statement just above checks for it.